e2e-tests/admin/tests/alias-ent.spec.js

```diff
@@ -101,6 +110,9 @@ test.describe('Aliases (Enterprise)', async () => {
       ).toBeVisible();
       await page.getByRole('button', { name: 'Dismiss' }).click();
     } finally {
+      if (connect) {
+        connect.kill('SIGTERM');
```

Comment:
This is fine but I just wanted to mention the [default](https://nodejs.org/api/child_process.html#subprocesskillsignal) for `kill()` is `SIGTERM`